src/pages/Home.js

```diff
@@ -27,6 +29,12 @@ const Home = () => {
     }
   }, [books]);
 
+  useEffect(() => {
+    if (!isAuthenticated || !user.email_verified) {
+      navigate("/onboarding");
+    }
+  }, []);
```

Comment:
The problems of user who have expired verify email need to be fixed before this features goes online.